torch_disc/torch_disc/csrc/disc_compiler/passes/cluster.cpp

```diff
@@ -77,7 +90,8 @@ void CastGraphInputsToTensor(const std::shared_ptr<Graph>& graph,
       sub_graph->replaceInput(i, cast_tensor->output());
 
       // TODO(Yancey1989): cast output
```

Comment:
TODO no longer valid?

Reply:
I think it's valid in the future, the output value in a disc module may be Scalar type, we should cast them to Tensor inner the module and cast them back to Scalar outer the module, but we have not met this scene, will fix this TODO in the future. And #176 introduced some background about the casting operation.

Reply:
I see